src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsallocations/service/GetUnallocatedCaseServiceTest.kt

```diff
@@ -29,8 +31,8 @@ internal class GetUnallocatedCaseServiceTest {
     )
     every { mockRepo.findByTeamCode("TM1") } returns listOf(unallocatedCaseEntity)
     every { mockRepo.existsById(id) } returns false
-    every { mockWorkforceAllocationsToDeliusApiClientClient.getDeliusCaseDetails(listOf(unallocatedCaseEntity)) } returns Flux.empty()
-    val cases = GetUnallocatedCaseService(mockRepo, mockk(), mockk(), mockWorkforceAllocationsToDeliusApiClientClient).getAllByTeam("TM1").collectList().block()
+    every { runBlocking { mockWorkforceAllocationsToDeliusApiClientClient.getDeliusCaseDetails(listOf(unallocatedCaseEntity)) } } returns emptyFlow()
```

Comment:
Hmm.. having `every .. runblocking` is confusing... I suppose if that's the way mockk() works then we just have to use it 🤷

Reply:
yeah, its how mockk() works, I'll check to see if there is a coroutines mockk out there

Reply:
looks like you can use `coEvery`